pkgs/applications/office/libreoffice/default-gen-shell.nix

```diff
@@ -19,9 +19,11 @@ stdenv.mkDerivation {
     builder = ./download-list-builder.sh;
   };
 
+  buildInputs = [ python36 ];
+
   shellHook = ''
     function generate {
-      ./generate-libreoffice-srcs.sh | tee libreoffice-srcs.nix
+      python3 generate-libreoffice-srcs.py > libreoffice-srcs.nix
```

Comment:
I thought the lesson learned from NixOS was that shebangs were somewhat evil and `/usr/bin/env` was just a least-painful compromise. No problem adding it if you like, though.